src/scenes/Question/scenes/Answer/Answer.jsx

```diff
@@ -46,19 +52,27 @@ class Answer extends Component {
     }
   }
 
-  cleanSources() {
-    const { sources } = this.state
-    return sources
-      .map(s => {
-        if (s.new) return omit(s, ['id', 'new'])
-        return s
+  sourcesToList(sources) {
```

Comment:
This method (and its inverse) could use better names I think :). It's not really clear what they do.

Reply:
I don't see better names for this. Do you have some suggestions?

Reply:
Well, I assume "source" is well-defined, but what about list? "sources" is already a list, so what is the destination format exactly. A list of what? Why does it drop the id if it's a uuid?

Reply:
What about sourcesToPairList & pairListToSources ?

Reply:
Yes that would fit. `sourcesToKeyValuePairs`/`keyValuePairsToSources` are even better I think.